### PR TITLE
Add ReqsLang package

### PR DESCRIPTION
**Pull request by petarslovic**
Add syntax highlighting for [ReqsLang](https://github.com/petarslovic/reqs-lang).

### CODE REVIEW

File: repository/r.json [hunk 1]
```diff
@@ -882,6 +882,17 @@
 			]
 		},
 		{
+			"name": "ReqsLang",
+			"details": "https://github.com/petarslovic/sublime-reqs-language",
+			"labels": ["requirements", "todo"],
```
Comment (wbond) [inline repository/r.json]: You probably want to add the `language syntax` label

Comment (petarslovic) [inline repository/r.json]: It is a syntax highlighting plugin for my own language, so not sure if I should add it as a label?

Comment (wbond) [inline repository/r.json]: Since it is a `.tmLanguage`, I think it makes sense.

Comment (petarslovic) [inline repository/r.json]: Right, cool, thanks for the feedback

Comment (wbond) [inline repository/r.json]: Sorry, I meant literally the label `language syntax`: https://packagecontrol.io/browse/labels/language%20syntax

Comment (petarslovic) [inline repository/r.json]: Ah, okay, got it now! :D

**Pull request merged**